wagtailvideos/templates/wagtailvideos/videos/index.html

```diff
@@ -22,8 +22,9 @@
 {% block content %}
     {% trans "Videos" as video_str %}
 
-    {% trans "Add a video" as add_video_str %}
-    {% include "wagtailadmin/shared/header.html" with title=video_str add_link="wagtailvideos:add_multiple" icon="media" add_text=add_video_str search_url="wagtailvideos:index" %}
+    {% trans "Add videos" as add_video_str %}
+    {% url "wagtailvideos:add_multiple" as action_url %}
```

Comment:
Was this left in by mistake? Doesn't look like `action_url` is used anywhere

Reply:
oops i didn't mean to push that with the pull request. will revert as it won't be backwards compatible. it wasn't a mistake, action_url is the variable used in the wagtailadmin/shared/header.html template so i did not include it in the 'with' parameters.